pom.xml

```diff
@@ -113,6 +114,14 @@
             <artifactId>sendgrid-java</artifactId>
         </dependency>
 
+        <!-- Email Validator -->
+        <dependency>
+            <groupId>commons-validator</groupId>
+            <artifactId>commons-validator</artifactId>
+            <version>${commonsvalidator.version}</version>
+        </dependency>
```

Comment:
javax.validation.constraints.Email perfectly executes that part of the job and it's already used in the project.